public/templates/_sidebar.tmpl

```diff
@@ -5,9 +5,17 @@
         <p class="menu-label">General</p>
         <ul class="menu-list">
             <li><a {{if eq $page "dashboard"}}class="is-active"{{end}} href="/internal">Dashboard</a></li>
-            <li><a {{if eq $page "users"}}class="is-active"{{end}} href="/internal/users">Users</a></li>
-            <li><a {{if eq $page "roles"}}class="is-active"{{end}} href="/internal/roles">Roles</a></li>
-            <li><a {{if eq $page "permissions"}}class="is-active"{{end}} href="/internal/permissions">Permissions</a></li>
+            {{if (memberAddPermission .UserID)}}
```

Comment:
Can this be `(checkPermission .UserID "ManageMembers.Members.Add")`?

Reply:
I think that was missed when I initially did that but yes, I have fixed it